manifests/init.pp

```diff
@@ -257,6 +257,7 @@
   Optional[Boolean] $analytics = undef,
   Optional[Boolean] $hide_guarded_distributions = undef,
   Optional[Integer[1,100]] $import_workers_percent = undef,
+  Optional[Boolean] $use_pulp2_content_route = false,
```

Comment:
Other than the lint comments, I'd rename it to indicate it's the file part. Though perhaps it should be `isos` now that `file` is no longer a plugin?
```suggestion
  Optional[Boolean] $use_pulp2_file_content_route = false,
```